packages/flame/lib/src/sprite.dart

```diff
@@ -97,18 +101,30 @@ class Sprite {
     Anchor anchor = Anchor.topLeft,
     Paint? overridePaint,
   }) {
-    final drawPosition = position ?? Vector2.zero();
-    final drawSize = size ?? srcSize;
-
-    final delta = anchor.toVector2()..multiply(drawSize);
-    final drawRect = (drawPosition - delta).toPositionedRect(drawSize);
+    if (position != null) {
+      _tmpRenderPosition.setFrom(position);
+    } else {
+      _tmpRenderPosition.setZero();
+    }
+
+    if (size != null) {
+      _tmpRenderSize.setFrom(size);
+    } else {
+      _tmpRenderSize.setFrom(srcSize);
+    }
```

Comment:
```suggestion
    _tmpRenderSize.setFrom(size ?? srcSize);
```

Reply:
Lol, really is time for me to go to bed :joy: 
Thanks!